lib/ide/src/executor/global.rs

```diff
@@ -0,0 +1,56 @@
+//! This module defines a shared, global state that stores a task spawner.
+//!
+//! It is expected that as a part of one-time initialization routine, IDE shall
+//! call `set_spawner` to define a global spawner. Once that is done, the whole
```

Comment:
Wouldn't `init_spawner` be better name here?

Reply:
After reading docs I was sure this should be set once. What are the use cases of setting it multiple times?